### PR TITLE
Updating Docker file and Shell Script.

### PR DESCRIPTION
**Pull request by nikksabhishek**
1. Adding Packages to dockerfile - libaio libnsl libc6-compat
2. Modifying shell to add the link (ld-linux-x86-64.so.2 to the lib/).

### CODE REVIEW

File: script.sh [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #!/bin/sh
 
-cd /lib 
+cd /lib
+ln -s /lib64/* /lib
```
Comment (Shrinidhikulkarni7) [inline script.sh:6]: This would break applications designed for /lib32.
Can you specify which specific files to link?

Comment (nikksabhishek) [reply script.sh:6]: Linking ld-linux-x86-64.so.2 to the lib/ location.

Comment (Shrinidhikulkarni7) [reply script.sh:6]: Awesome, please update the pull request accordingly.
Thanks.

**Review by nikksabhishek — COMMENTED** [review]
Added comment in script.sh for user to identify link required for a different version.

**Review by Shrinidhikulkarni7 — APPROVED** [review]

**Pull request merged**